ldes-fragmentisers/ldes-fragmentisers-reference/README.MD

```diff
@@ -0,0 +1,74 @@
+# Reference fragmentation
```

Comment:
not a big fan of copying the docs in here as well.. would only use docs instead of here.

Reply:
I will remove the it for the other fragmentations as well then